### ISSUE TITLE
Add local classifier example

### ISSUE THREAD

**Issue by ryantibs** [cmu-delphi/epiprocess#32]
Currently listed as a TODO at the [bottom of the percent change vignette](https://cmu-delphi.github.io/epiprocess/articles/pct_change.html#building-and-running-a-local-classifier-1).

My thinking is that we could do the analogous thing to what we did for the local forecaster at the [bottom of the slide vignette](https://cmu-delphi.github.io/epiprocess/articles/slide.html#building-and-running-a-local-forecaster-1).

**Comment by ryantibs** [comment]
Updated thinking on this. We could do the following:

- Just consider national data, over a time period that includes a couple of waves (say winter wave of 20-21, and delta wave).
- Predict upswings in deaths, where percentage change is >= 10% (or some other threshold that passes sanity checks for being reasonable).
- Consider two classifiers: one that uses lagged percentage change values of deaths, and one that uses this as well as lagged percentage change values of cases.
- Run them using `epi_slide()` and compare them in some very simple way (could even be qualitatively).
- Recall none of this uses properly-versioned data. That is a job for `epi_archive` and the `epipredict` package.  We should reference these tools in the vignette (or leave a placeholder for reference this until we figure out what exactly it should point to, as done in the slide vignette).

**Comment by ryantibs** [comment]
Awesome, thanks @qpmnguyen!

**Comment by ryantibs** [comment]
@qpmnguyen Have you started this yet? I'm thinking now that the best thing to do might be to scrap this example and combine this percentage change vignette with the next one, on estimating derivatives, and make it all under the theme of local growth rates.

I wasn't very happy with the estimating derivatives vignette anyway (as I commented in the next issue #33) so I think this will make sense on its behalf as well. And I would want to figure out how to implement centered derivatives in a fluid way before updating the vignettes, so it'd be blocked on #33.

**Comment by qpmnguyen** [comment]
I haven't started this yet. So I am assuming this is tabled and merged into the estimating derivatives vignette until issue #33 is resolved?

**Comment by ryantibs** [comment]
Yes I think that's the right approach. And feel free to ping me on slack if I can be helpful guiding you in looking for things to do, or write a message to the tooling channel in general.

**Comment by ryantibs** [comment]
Closed indirectly via #40, as I implemented growth rates and merged the derivatives vignette into the growth rate one (and no need for this example anymore).
